Makefile.am

```diff
@@ -32,7 +32,7 @@
 # GLOBAL SETTINGS
 ACLOCAL_AMFLAGS = -I m4
 AM_CPPFLAGS += -I$(top_srcdir)/src \
-               -DGEOPM_DEFAULT_PLUGIN_PATH=\"$(pkglibdir)\" \
```

Comment:
Since the base package name is geopm, I think this change should resolve to the same string?  Perhaps it is clearer with the change?

Reply:
Yes, it does resolve to the same string.  This was done so that it's obvious where this is pointing when you compare this against the same line in ```service/Makefile.am```.  Previously this was broken in the service Makefile.